storagenode/gracefulexit/chore.go

```diff
@@ -0,0 +1,113 @@
+// Copyright (C) 2019 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package gracefulexit
+
+import (
+	"context"
+	"sync"
+	"time"
+
+	"go.uber.org/zap"
+	"gopkg.in/spacemonkeygo/monkit.v2"
+
+	"storj.io/storj/internal/sync2"
+	"storj.io/storj/storagenode/satellites"
+)
+
+var mon = monkit.Package()
+
+// Chore checks for satellites that are being exited and creates a worker per satellite to complete the process.
```

Comment:
I think it could help with clarity if this comment were changed to "Chore checks for satellites that the node is exiting and creates..."

Reply:
updated